.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [About the WooCommerce REST API](#about-the-woocommerce-rest-api)
+- [API Versioning and how it affects development](#api-versioning-and-how-it-affects-development)
+- [Adding new endpoints](#adding-new-endpoints)
+  - [Schema first development](#schema-first-development)
+- [Deprecation of older versions](#deprecation-of-older-versions)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## About the WooCommerce REST API
+
+WooCommerce is fully integrated with the [WordPress REST API](https://developer.wordpress.org/rest-api/). This allows WooCommerce data to be created, read, updated, and deleted using requests in JSON format and using various WordPress REST API Authentication methods with standard HTTP verbs which are understood by most HTTP clients.
+
+To read more about usage, [see our getting started guide here](https://github.com/woocommerce/woocommerce/wiki/Getting-started-with-the-REST-API).
+
+The API in WooCommerce ships with WooCommerce core, so each instance of the API is running on a user's store/server. **This is important** because it means clients could be running on different versions which you need to account for in your application. We use API versioning to help with this.
+
+## API Versioning and how it affects development
+
+Core hosts multiple versions if the API for backwards compatibility and so apps can continue to use the same API version after updates to target as many of their users as possible.
+
+Accessing a certain version is done in the URL. For example, to target `v1` of the Rest API you can call:
+
+```
+/wp-json/wc/v1
+```
+
+And to access `v2`:
+
+```
+/wp-json/wc/v1
```

Comment:
Sounds like need to update to `/wp-json/wc/v2`.

Reply:
:+1: